lib/Monads/reader_option/Reader_Option_Monad.thy

```diff
@@ -198,7 +198,7 @@ definition obind :: "('s,'a) lookup \<Rightarrow> ('a \<Rightarrow> ('s,'b) look
 
 (* Enable "do { .. }" syntax *)
 adhoc_overloading
-  Monad_Syntax.bind obind
+  Monad_Syntax.bind \<rightleftharpoons> obind
```

Comment:
This change seems to indicate that you need to spell out the notation both here and at the definition point of obind, right? I'm confused by this.

Reply:
it's only the syntax of `adhoc_overloading` that changed. It wants a `⇌` now, no other changes than that really.